doc/langref.html.in

```diff
@@ -7650,7 +7650,7 @@ fn List(comptime T: type) type {
       {#header_open|@truncate#}
       <pre>{#syntax#}@truncate(comptime T: type, integer: var) T{#endsyntax#}</pre>
       <p>
-      This function truncates bits from an integer type, resulting in a smaller
+      This function truncates bits from an integer type (or vector of integers), resulting in a smaller
```

Comment:
It's not obvious if you mean it truncates the vector as a whole, or each element in a vector.

Reply:
good point